Code/GraphMol/DistGeomHelpers/Wrap/rdDistGeom.cpp

```diff
@@ -215,6 +216,40 @@ void setBoundsMatrix(DGeomHelpers::EmbedParameters *self,
       new DistGeom::BoundsMatrix(nrows, sdata));
 }
 
+python::tuple getExpTorsHelper(const RDKit::ROMol &mol,
+                               bool useExpTorsions = false,
```

Comment:
It seems odd that useExpTorsions defaults to false.

What does this mean in this context?  The python wrapper defaults to True I think.

Reply:
> It seems odd that useExpTorsions defaults to false.
> 
> What does this mean in this context? The python wrapper defaults to True I think.

This is just the helper function which is called by the python wrappers. The default values here don't mean anything.
It is confusing though, so I will remove them.